docs/modules/traits/pages/registry.adoc

```diff
@@ -1,8 +1,9 @@
 = Registry Trait
 
+// Start of autogenerated code - DO NOT EDIT! (badges)
+image:https://img.shields.io/badge/2.3.0?label=Deprecated&labelColor=%23C40C0C&color=gray[Deprecated Badge]
```

Comment:
Are you sure this is the correct format? in the rich diff I cannot see the image, but the text instead. We are using this badge for coverage as well, for example: `image:https://img.shields.io/badge/Coverage-38.0%25-yellow.svg["Go coverage"]`

Reply:
I think the format for the SVG is `https://img.shields.io/badge/Deprecated-2.3.0-orange.svg`

Reply:
It renders to this html

```

<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="27" height="20" role="img" aria-label="foo"><title>foo</title><linearGradient id="s" x2="0" y2="100%"><stop offset="0" stop-color="#bbb" stop-opacity=".1"/><stop offset="1" stop-opacity=".1"/></linearGradient><clipPath id="r"><rect width="27" height="20" rx="3" fill="#fff"/></clipPath><g clip-path="url(#r)"><rect width="0" height="20" fill="#007ec6"/><rect x="0" width="27" height="20" fill="#007ec6"/><rect width="27" height="20" fill="url(#s)"/></g><g fill="#fff" text-anchor="middle" font-family="Verdana,Geneva,DejaVu Sans,sans-serif" text-rendering="geometricPrecision" font-size="110"><text aria-hidden="true" x="135" y="150" fill="#010101" fill-opacity=".3" transform="scale(.1)" textLength="170">foo</text><text x="135" y="140" transform="scale(.1)" fill="#fff" textLength="170">foo</text></g></svg>
```